spec/acceptance/peadm_spec/plans/test_migration.pp

```diff
@@ -2,6 +2,7 @@
   String $primary_host,
   String $new_primary_host,
   Optional[String] $new_replica_host = undef,
+  Optional[String] $primary_postgresql_host = undef,
```

Comment:
shouldnt need to add 'primary_postgresql_host' as we already have new_primary_postgresql_host